src/extension/package.json

```diff
@@ -49,6 +49,13 @@
         "category": "%webTemplateStudioExtension.commands.wts%"
       }
     ],
+    "keybindings": [
+      {
+        "command": "webTemplateStudioExtension.wizardLaunch",
+        "key": "ctrl+shift+a",
```

Comment:
I would consider leaving this blank so that it would appear in the keyboard shortcuts view but up to individual users to define: 
![image](https://user-images.githubusercontent.com/23327878/61010003-7f863800-a329-11e9-8c2a-b587167004cf.png)

Reply:
So you think its better to just give the user the ability but define no default? If so the extension already provides the ability. This PR just defines a default for the user.